### PR TITLE
Update KeePassOTP.pt.language.xml

### PR DESCRIPTION
**Pull request by Havokdan**

### CODE REVIEW

**Comment by Rookiestyle** [comment]
If this is the new version, please also update the version tag.
You can push this to your patch-3 branch

File: Translations/KeePassOTP.pt.language.xml [hunk 1]
```diff
@@ -1,174 +1,174 @@
...
-  </item>
-  <item>
-    <key>OTP_OpenDB</key>
-    <value>Abra o banco de dados OTP - {0}</value>
-  </item>
-  <item>
-    <key>OTP_CreateDB_PWHint</key>
+<?xml version="1.0" encoding="utf-8"?>
+<Translation>
+  <TranslationVersion>2</TranslationVersion>
```
Comment (Rookiestyle) [inline Translations/KeePassOTP.pt.language.xml:3]:
```suggestion
  <TranslationVersion>3</TranslationVersion>
```

Please also update the version tag

**Review by Rookiestyle — CHANGES_REQUESTED** [review]

**Comment by Havokdan** [comment]
I don't really understand how github works...

**Comment by Rookiestyle** [comment]
Thanks!

**Pull request merged**